dear_petition/petition/tests/factories.py

```diff
@@ -99,8 +99,8 @@ class OffenseRecordFactory(factory.DjangoModelFactory):
     law = "20-141(J1)"
     code = "4450"
     action = CHARGED
-    severity = "TRAFFIC"
-    description = "SPEEDING(96 mph in a 70 mph zone) "
+    severity = "MISDEMEANOR"
+    description = "RESISTING PUBLIC OFFICER"
```

Comment:
So, was this law `20-141(J1)` code `4450` just incorrectly linked to speeding, when it should have been resisting an officer? Just checking because the resisting part isn't named in the description and I want to be sure it is a relevant change here.

Reply:
Ahh, good catch! I should update `law` and `code` to match `MiSDEMEANOR`. 
This is a test record and since `Traffic` is excluded now, I had to update it with a severity other than "TRAFFIC", but I missed these properties. Will update those shortly

Reply:
Oh, that's not a big deal — I just completely missed the fact that this was test data!